packages/tpetra/core/src/Tpetra_CrsMatrix_def.hpp

```diff
@@ -4208,6 +4227,15 @@ namespace Tpetra {
                                 x_lcl_1d, false, false);
     }
     else {
+      // FENCE REVIEW - NOT TESTED
+      //   Testing: This code is NOT exercised by unit tests.
+      //   GTX960:  N/A
+      //   White:   N/A
+      //   Plan:    Potentially add unit testing for this code.
+      //            Potentially remove this fence but following code appears
+      //            to need refactoring.
+      //   Notes:
+
```

Comment:
@kddevin Same as above. We need this fence or the fence at the end of the routine. This can be a discussion topic.

Reply:
Removed with PR 7560